test/box-luatest/pagination_test.lua

```diff
@@ -0,0 +1,801 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local tree_g = t.group('Tree index tests',
+    t.helpers.matrix{engine={'memtx'}, disable_ffi={true, false}})
```

Comment:
Please move this test to `engine-luatest`. We keep all multi-engine tests there for some reason. (Maybe, we shouldn't?)

Reply:
I moved the test to `engine-luatest`